code/modules/security/workshop.dm

```diff
@@ -83,6 +83,3 @@
 	else
 		say("Workshop shutdown underway! Standby for reboot...")
 
-/obj/machinery/computer/holodeck/prison/load_program()
-	. = ..()
-	offline = FALSE
```

Comment:
When I mentioned 'load_program' being useless, I was actually refering to the call in ui_act rather than this proc